### PR TITLE
update to docs to distinguish between a catalog and catalog file

### PR DESCRIPTION
**Pull request by gwbischof**

### CODE REVIEW

File: docs/source/glossary.rst [hunk 1]
```diff
@@ -19,6 +19,9 @@ Glossary
         server or another third-party data service, like a SQL database. Thus, catalogs form a hierarchy: any
         catalog can contain other, nested catalogs.
 
+    Catalog file
+        A specification file which contains a list of named entries describing how to load data sources.
```
Comment (martindurant) [inline docs/source/glossary.rst]: Would you mind specifying that this is in YAML format and link to the docs where that syntax is described?

**Comment by gwbischof** [comment]
I think this part of docs would be a bit clearer if `Catalog` was replaced by `Catalog file`
https://intake.readthedocs.io/en/latest/quickstart.html#opening-a-catalog

File: docs/source/glossary.rst (hunk 1 above)
Comment (gwbischof) [reply docs/source/glossary.rst]: Sure i'll add that now

**Comment by gwbischof** [comment]
Also fixed a broken link to CSVSource here: https://intake.readthedocs.io/en/latest/quickstart.html#opening-a-catalog

**Comment by martindurant** [comment]
OK, that's great, thank you.

**Pull request merged**